src/JSONAPIExtension.php

```diff
@@ -63,11 +65,41 @@ public static function getSubscribedEvents()
             KernelEvents::EXCEPTION => [
                 ['error', 515],
             ],
+            KernelEvents::CONTROLLER => [
+                ['disableFrontend', -1025]
```

Comment:
This is massively high (late in terms of priority, but not the point :stuck_out_tongue: ). Does it really need to be?